docs/storage-schema.md

```diff
@@ -12,6 +12,7 @@ Heapster exports the following metrics to its backends.
 | cpu/request | CPU request (the guaranteed amount of resources) in millicores. |
 | cpu/usage | Cumulative CPU usage on all cores. |
 | cpu/usage_rate | CPU usage on all cores in millicores. |
+| cpu/load | CPU load. |
```

Comment:
It would be great to add a bit more information about units of this metric, because I had to check cadvisor documentation to understand that it is actually multiplied by 1000.

Reply:
Will do.

Reply:
@loburm Done. PTAL.